ci/cpu/clang14_release_stdexec.yml

```diff
@@ -0,0 +1,30 @@
+include:
+  - local: 'ci/common-ci.yml'
+
+cpu clang14 stdexec release deps:
+  extends: .build_deps_common
+  variables:
+    EXTRA_APTGET: "clang-14 libomp-14-dev"
+    COMPILER: clang@14
```

Comment:
@msimberg stdexec package contains: `conflicts("%clang@:13")`

Reply:
That's wrong :disappointed: let's go with clang 14 here then. We can update the conflict separately in spack.

Reply:
Update: the constraint in the stdexec package file is subtly wrong, but intentionally so. The issue is that clang prior to version 14 does not support coroutines correctly when used together with libstdc++ (see e.g. https://github.com/facebook/folly/pull/1694#issuecomment-1157255386). Since we can't model the standard library (and its version) in spack, the spack package uses the conservative option and just requires clang 14 or newer.